### PR TITLE
Add agent arguments and filter test classpath

### PR DESCRIPTION
**Pull request by bantonsson**

### CODE REVIEW

File: maxwell/src/main/java/maxwell/Maxwell.java [hunk 1]
```diff
@@ -1,9 +1,15 @@
 package maxwell;
 
 import java.lang.instrument.Instrumentation;
+import java.util.regex.Pattern;
 
 public class Maxwell {
     public static void premain(String agentArgs, Instrumentation instrumentation) {
-        System.out.println("Agent 86");
```
Comment (henrikengstrom) [inline maxwell/src/main/java/maxwell/Maxwell.java]: 👍

**Comment by henrikengstrom** [comment]
LGTM.

**Comment by ironfish** [comment]
nice

**Comment by pvlugter** [comment]
Awesome. Looks good!

File: maxwell/src/main/java/maxwell/Maxwell.java (hunk 1 above)
Comment (pvlugter) [inline maxwell/src/main/java/maxwell/Maxwell.java]: I'm curious. What's the :+1: for here?

Comment (henrikengstrom) [inline maxwell/src/main/java/maxwell/Maxwell.java]: That the system out has been moved down a couple of lines... :)

Comment (pvlugter) [inline maxwell/src/main/java/maxwell/Maxwell.java]: :smile:

**Pull request merged**